src/pages/HomePage.js

```diff
@@ -1,12 +1,20 @@
 import React, { Component } from 'react'
 import { useNavigate } from 'react-router-dom';
+import logoutRequest from '../services/LogoutRequest';
 
 class HomePage extends Component {
+
+  constructor(props){
+    super(props);
+}
   
   logoutUser = () => {
     window.localStorage.setItem('blogSiteUserLoggedIn', false);
-    this.props.changeLoginState(false);
-    this.props.navigator('/login');
+    logoutRequest(() => {
+      this.props.changeLoginState(false);
+      this.props.navigator('/login');
+    }).then();
```

Comment:
Did you verify it works?

Reply:
Apologies, I thought i had made this change in this PR. I made it in the GET request PR. Changes made and uploaded.